src/couch/src/couch_server.erl

```diff
@@ -823,7 +823,7 @@ get_default_engine(Server, DbName) ->
         engines = Engines
     } = Server,
     Default = {couch_bt_engine, make_filepath(RootDir, DbName, "couch")},
-    case config:get("couchdb", "default_engine") of
+    case config:get("couchdb", "default_engine", ?DEFAULT_ENGINE) of
```

Comment:
You  don't need case when you use `config:get/3` you would always receive a string (assuming default value is string, which is true in your case).

Reply:
Changed.